src/core/server/bootstrap.ts

```diff
@@ -68,10 +65,13 @@ export async function bootstrap({
     isDevClusterMaster: isMaster && cliArgs.dev && features.isClusterModeSupported,
   });
 
-  const rawConfigService = new RawConfigService(
-    env.configs,
-    rawConfig => new LegacyObjectToConfigAdapter(applyConfigOverrides(rawConfig))
-  );
+  const rawConfigService = new RawConfigService(env.configs, rawConfig => {
+    if (features.stripXpackConfig) {
+      delete rawConfig.xpack;
```

Comment:
I noticed we use [unset](https://github.com/restrry/kibana/blob/master/src/legacy/utils/unset.js#L23) in [other places](https://github.com/restrry/kibana/blob/2a1493169d5e58a7112f3870067d1f4a7840d3e2/src/legacy/server/config/config.js#L70-L71)

Reply:
Since I've moved this out of the new platform and lodash 3 doesn't have an unset method I've just left the delete for now.